mobile.css

```diff
@@ -0,0 +1,580 @@
+}
+
+.intro .p1 {
+  padding-bottom: 2px;
+}
+
+.timeZone {
+  width: 20rem;
+  color: #272a31;
+  text-align: center;
+  justify-content: center;
+  margin: 0 auto;
+  margin-top: 40px;
+}
+
+.timeZone h1 {
+  font-size: 20px;
+  margin-bottom: 10px;
+}
+
+.timeZone .p3 {
+  font-size: 10px;
+}
+
+.CreativeCmn {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 20rem;
+  height: 23rem;
+  color: #272a31;
+  text-align: center;
+  justify-content: center;
+  margin: 0 auto;
+  margin-top: 40px;
+}
+
+.CreativeCmn h1 {
+  font-size: 20px;
+  margin-bottom: 10px;
+}
+
+.CreativeCmn .p3 {
+  font-size: 10px;
+}
+
+.anchor a {
+  color: #272a31;
+}
+
+.Logo {
+  height: 150px;
+}
+
+.logo-div {
+  border: rgb(112, 112, 112) solid 1px;
+  padding-left: 2rem;
+  padding-right: 2rem;
+}
+
+.line {
+  display: block;
+  width: 40px;
+  justify-content: center;
+  margin: 0 auto;
+  border: 1px solid rgb(255, 82, 82);
+}
+
+.SummitCmn {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 20rem;
+  height: 40rem;
+  color: #272a31;
+  text-align: center;
+  justify-content: center;
+  margin: 0 auto;
+  margin-top: 40px;
+}
+
+.SummitCmn h1 {
+  font-size: 20px;
+}
+
+.image1 {
+  background-image: url(./icon/teacher.jpg);
+  height: 10rem;
+  color: #45474b;
+  background-size: 22rem;
+  font-weight: bold;
+  font-size: 18px;
+  padding-top: 30px;
+}
+
+.image2 {
+  background-image: url(./icon/classPic.jpg);
+  height: 10rem;
+  color: #45474b;
+  background-size: 22rem;
+  font-weight: bold;
+  font-size: 18px;
+  padding-top: 30px;
+}
+
+.foot-sctn1 {
+  background-color: #272a31;
+}
+
+.foot-title {
+  width: 70px;
+  justify-content: center;
+  margin: 0 auto;
+  padding-top: 20px;
+}
+
+.foot-ptnr {
+  /* font-weight: 500; */
+  color: #5d5f63;
+}
+
+.foot-border {
+  border-bottom: #ec5242 solid 1px;
+  display: block;
+  justify-items: center;
+  margin: 0 auto;
+  width: 50px;
+}
+
+.net {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  font-size: 20px;
+  width: 90%;
+}
+
+.net h2 {
+  color: #5d5f63;
+}
+
+.nets {
+  display: flex;
+  justify-items: center;
+  margin: 0 auto;
+  justify-content: space-around;
+  font-size: 15px;
+  width: 70%;
+}
+
+.nets h2 {
+  color: #5d5f63;
+}
+
+.naver {
+  font-weight: 1000;
+}
+
+.ggl {
+  font-weight: 100;
+}
+
+.foot-sctn2 {
+  display: flex;
+  justify-content: space-between;
+  justify-items: center;
+  margin: 0 auto;
+  padding: 5rem 0;
+  width: 60%;
+}
+
+.foot-sctn2 .common {
+  width: 70%;
+}
+
+.foot-sctn2 img {
+  height: 5rem;
+}
+
+footer .footer {
+  display: flex;
+  width: 80%;
+  gap: 20px;
+  justify-content: center;
+  margin: 0 auto;
+  margin: 20px;
+  padding: 20px;
+}
+
+@media (max-width: 768px) {
+  html,
+  body {
+    padding: 0;
+    margin: 0;
+    overflow-x: hidden;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  }
+
```

Comment:
- [ ] Great job using media queries, however, it's one of the project requirements to build a mobile-first website, which means the styling should be first for mobile and we use media queries for other sized devices. Please fix that.